packages/cni-plugins/Cargo.toml

```diff
@@ -14,3 +14,4 @@ sha512 = "24e8fcedbff2ae7a83aa96085b546b164de6a0884d593e3b5386e9d2de3c4d9a215db9
 
 [build-dependencies]
 glibc = { path = "../glibc" }
+iptables = { path = "../iptables" }
```

Comment:
Would it work to make these regular `dependencies`, and to use that instead of `build-dependencies` for the variant also?

That might be more clear. Otherwise we lose some information here about the `BuildRequires` vs `Requires` distinction.

Reply:
Yeah, that makes sense. It also 'relaxes' our instructions to Cargo about the build graph to something more representative of reality.